pkg/controller/v1beta1/inferenceservice/controller_test.go

```diff
@@ -32,6 +33,37 @@ import (
 	"time"
 )
 
+func createTestInferenceService(serviceKey types.NamespacedName, hasStorageUri bool) *v1beta1.InferenceService {
+
+	predictor := v1beta1.PredictorExtensionSpec{
+		Container: v1.Container{
+			Name: "kfs",
+		},
+	}
+	if hasStorageUri {
+		storageUri := "s3://test/mnist/export"
+		predictor.StorageURI = &storageUri
```

Comment:
You can use proto.string("  ") which will give you a string pointer instead of saying this in two lines. There's also aws.string and a number of other libraries that will do this for you :). I think this repo already uses proto.